lib/service/proxy_settings.go

```diff
@@ -46,6 +46,12 @@ type proxySettings struct {
 	accessPoint networkConfigGetter
 }
 
+// GetOpenAIAPIKey returns the OpenAI API key.
+// TODO(jakule): Remove once plugin support is added to OSS.
```

Comment:
do we need to put a note for this in an issue somewhere do we don't forget? iirc we're not great at keeping a lot of the todos we put in up to date/fixed/taken care of